### PR TITLE
ephemeralpg: 3.1 -> 3.3, fix for nixos-update's runner

### PR DESCRIPTION
**Pull request by medv**
## Description of changes

Relevant issue where I was instructed to submit this PR: https://github.com/ryantm/nixpkgs-update/issues/380

The old homepage for this project was replaced some time ago, breaking @r-ryantm and resulting in a very outdated version of this excellent package in nixpkgs.

## Things done

1. Updated homepage from http://ephemeralpg.org to https://eradman.com/ephemeralpg which is the official link from the project's [repository](https://github.com/eradman/ephemeralpg/).
2. Upgraded package from 3.1 to 3.3
3. Added myself to maintainers? Unsure if I had to do that, I see it often asked in other PRs

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/tools/database/ephemeralpg/default.nix [hunk 1]
```diff
@@ -1,10 +1,10 @@
 { lib, stdenv, fetchurl, postgresql, getopt, makeWrapper }:
 stdenv.mkDerivation rec {
   pname = "ephemeralpg";
-  version = "3.1";
+  version = "3.3";
   src = fetchurl {
-    url = "http://ephemeralpg.org/code/${pname}-${version}.tar.gz";
-    sha256 = "1ap22ki8yz6agd0qybcjgs4b9izw1rwwcgpxn3jah2ccfyax34s6";
+    url = "https://eradman.com/ephemeralpg/code/${pname}-${version}.tar.gz";
+    sha256 = "02mf0zw7lmb8c9d3fvjb17zikgdi36aaa53h8m17q9kh59yjnm55";
```
Comment (wegank) [inline pkgs/development/tools/database/ephemeralpg/default.nix:7]:
```suggestion
    hash = "sha256-pVQrfSpwJnxCRXAUpZQZsb0Z/wlLbjdaYmhVevgHrgo=";
```

**Comment by wegank** [comment]
@ofborg build ephemeralpg ephemeralpg.passthru.tests

**Comment by medv** [comment]
How did you calculate that new hash? I did `nix-hash --type sha256 --base32 --flat` resulting in a hash compatible with the previously stated version, unsure what my other options were heh

**Review by wegank — APPROVED** [review]

**Comment by wegank** [comment]
`nix store prefetch-file https://eradman.com/ephemeralpg/code/ephemeralpg-3.3.tar.gz`

**Comment by Artturin** [comment]
Please squash the `fix hash` commit to the update commit

Tried to do it myself but got `! [remote rejected]           patch-1 -> patch-1 (permission denied)` even though there should be no permission issue

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**